api/app/signals/apps/questionnaires/rest_framework/fields.py

```diff
@@ -40,14 +40,19 @@ class QuestionnairePublicHyperlinkedIdentityField(HyperlinkedRelatedFieldMixin,
     lookup_field = 'uuid'
 
     def to_representation(self, value):
-        return OrderedDict([('self', dict(href=self._get_url(value, 'public-questionnaire-detail'))), ])
+        return OrderedDict([
+            ('curies', dict(name='sia', href=self.reverse('signal-namespace', request=self.context.get('request')))),
+            ('self', dict(href=self._get_url(value, 'public-questionnaire-detail'))),
+            ('sia:create-session', dict(href=self._get_url(value, 'public-questionnaire-create-session'))),
+        ])
 
 
 class QuestionHyperlinkedIdentityField(HyperlinkedRelatedFieldMixin, serializers.HyperlinkedIdentityField):
     lookup_field = 'retrieval_key'
 
     def to_representation(self, value):
         return OrderedDict([
+            ('curies', dict(name='sia', href=self.reverse('signal-namespace', request=self.context.get('request')))),
```

Comment:
I hope our frontend is not failing on extra information in de Question (and Session below) serialization.

Reply:
🤞🏼